### PR TITLE
Home issue

### PR DESCRIPTION
**Pull request by Suhanshu123**
I changed the code @immadisairaj sir acc. to what  u say. Now loading starts only after clickiing the button.

### CODE REVIEW

**Comment by immadisairaj** [comment]
Why is the Delay ?

File: app/src/main/java/com/example/immadisairaj/quiz/question/Question.java [hunk 1]
```diff
@@ -35,9 +35,10 @@
     public List<String> optC;
     public List<String> optD;
     public List<Integer> Answer;
+    public int check;
 
     public void setQuestion(Context context, Activity activity) {
-
+        check = 0;
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/quiz/question/Question.java]: Any use of 0.
Or you can use boolean

**Comment by Suhanshu123** [comment]
I'm making delay so that my object ' qAndq'  which I'm passing to QuizActivity is having least chance of null.
I have tried all way... it is working good on 10 second delay. and 'check' is used to  check the status of result from API whether result is successful,unsuccessful or still loading. We aren't using Boolean because I have to use three cases. I.e. 0,1,2

File: app/src/main/java/com/example/immadisairaj/quiz/HomeActivity.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home2);
+        view = findViewById(R.id.constraint_layout);
+
+    }
+
+    public void goNext(final View view) {
+        if (internetCheck() == true) {
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/quiz/HomeActivity.java]: No need of "_== true_"

**Pull request closed without merging**